bigquery_etl/usage/__main__.py

```diff
@@ -0,0 +1,20 @@
+import pathlib
+import yaml
+
+from jinja2 import Template
+
+from bigquery_etl.format_sql.formatter import reformat
+
+
+HERE = pathlib.Path(__file__).parent.absolute()
+ROOT = pathlib.Path(__file__).parent.parent.parent.absolute()
+
+
+with (HERE / "config" / "measures.yaml").open("r") as f:
+    data = yaml.safe_load(f)
+
+with (HERE / "templates" / "view_last_seen.sql").open("r") as f:
+    template = Template(f.read())
+
+for source in data["sources"]:
+    print(reformat(template.render(**source)))
```

Comment:
I'm guessing this will be similar to what we do for the Airflow DAG generation in that we require users that add new measures to generate the query? And then have a similar process for the CI and running checks and dry runs for making sure the generated queries are valid?

Reply:
Yes, currently this sort of approach would rely on PR authors to run generation locally. I am definitely hesitant to add that complexity without a clear approach to CI, documentation, etc.